src/enc/pm_tables.c

```diff
@@ -1375,6 +1375,7 @@ PRISM_ENCODING_TABLE(ibm857)
 PRISM_ENCODING_TABLE(ibm860)
 PRISM_ENCODING_TABLE(ibm861)
 PRISM_ENCODING_TABLE(ibm862)
+PRISM_ENCODING_TABLE(ibm863)
```

Comment:
Whoops now we're just missing the `ibm863` table. If you generate it with `bin/encodings ibm863` you should be able to copy-paste.